3.7/release-notes-new-features37.md

```diff
@@ -680,10 +852,31 @@ This change simplifies the installation procedures and internal code paths.
 Internal changes
 ----------------
 
-### Upgrade of bundled RocksDB library version
+### Upgraded bundled RocksDB library version
 
 The bundled version of the RocksDB library has been upgraded from 6.2 to 6.8.
 
+### Upgraded bundled OpenLDAP library version
+
+The OpenLDAP version used for the LDAP integration in the ArangoDB Enterprise 
+Edition has been upgraded to 2.4.50.
+This change has been backported to ArangoDB v3.6.5 as well.
+
+### Added libunwind library dependency
+
+The Linux builds of ArangoDB now use the third-party library
+[libunwind](https://github.com/libunwind/libunwind){:target="_blank"} to get
+backtraces and symbolize stack frames.
```

Comment:
?
```suggestion
backtraces and symbolized stack frames.
```

Reply:
No, it should be "symbolize" here, as it should be a verb and not an adjective.
I will add an extra "to" here to make it more clear.